src/index.js

```diff
@@ -0,0 +1,148 @@
+import './style.css';
+import reload from './assets/reload.svg';
+import entericonss from './assets/EnterIcon.svg';
+// import data from './dataArray.js';
```

Comment:
- [ ] It is unprofessional to leave commented code in your codebase, please kindly remove this line of code if it is not useful in your project 👍